modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java

```diff
@@ -159,25 +145,38 @@ public void testOIDCLogout() throws IOException {
     }
 
     private void createOIDCApplication() throws Exception {
+        ApplicationModel application = new ApplicationModel();
```

Comment:
Shall we fix the formatting issue by adding the missing new line?

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096